applications/source/zeromq-source/pom.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.springframework.cloud.stream.app</groupId>
+        <artifactId>stream-applications-core</artifactId>
+        <version>3.0.0-SNAPSHOT</version>
+        <relativePath/>
+    </parent>
+
+    <artifactId>zeromq-source</artifactId>
+    <version>3.0.0-SNAPSHOT</version>
+    <name>zeromq-source</name>
+    <description>0MQ source apps</description>
+    <packaging>jar</packaging>
+
+    <properties>
+        <zeromq.version>0.5.2</zeromq.version>
+    </properties>
+
+    <dependencies>
+
+        <!-- Spring Cloud Function dependencies -->
+        <dependency>
+            <groupId>org.springframework.cloud.fn</groupId>
+            <artifactId>zeromq-supplier</artifactId>
+            <version>1.0.0-SNAPSHOT</version>
+        </dependency>
+
+        <!-- Test dependencies -->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.junit.vintage</groupId>
+                    <artifactId>junit-vintage-engine</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-json</artifactId>
```

Comment:
Do we really do anything with JSON in tests for this project?

Reply:
I was thinking this as well. But when I removed it, it didn't configure the ConfigurationProperties correctly... at least in the test scenario.